src/backend/metal/src/lib.rs

```diff
@@ -282,8 +282,6 @@ impl Instance {
             // If the main layer is not a CAMetalLayer, we create a CAMetalLayer sublayer and use it instead.
             // Unlike on macOS, we cannot replace the main view as UIView does not allow it (when NSView does).
             let new_layer: CAMetalLayer = msg_send![class, new];
-            let bounds: CGRect = msg_send![view, bounds];
-            msg_send![new_layer, setBounds: bounds];
 
             let frame: CGRect = msg_send![view, frame];
             msg_send![new_layer, setFrame: frame];
```

Comment:
I believe this is more correct as `[new_layer setFrame: [main_layer bounds]]`. Frames are always in the parents coordinate space, and `bounds` are in view/layer's own coordinate space.

Reply:
Maybe `[CALayer setAutoresizingMask: kCALayerWidthSizable | kCALayerHeightSizable]` is an alternate solution to resizing in build swapchain? I haven't tested it, but the docs look good.

Reply:
Ahh `autoresizingMask` is macos only.

Reply:
:+1: moved to https://github.com/gfx-rs/gfx/issues/2938

Reply:
I've now addressed this here for iOS. We should handle #2938 for macOS in a separate PR.